Spelling corrections in binary code
Patch author: Ryan Finnie <ryan@finnie.org>

This patch corrects misspelled words in binaries, as found by lintian.  
Note, not all corrections (code comments, etc) were made, just binary 
visible words.  This patch will be forwarded upstream, with a 
recommendation that all spelling errors be fixed.

Currently found:
  * Initalize -> Initialize
  * seperated -> separated
  * arguements -> arguments
  * GNU Public License -> GNU General Public License

diff -ruN digitemp-3.6.0.dfsg.1.orig/digitemp.1 digitemp-3.6.0.dfsg.1/digitemp.1
--- digitemp-3.6.0.dfsg.1.orig/digitemp.1	2008-08-28 14:11:50.000000000 -0700
+++ digitemp-3.6.0.dfsg.1/digitemp.1	2012-12-08 13:54:27.997166589 -0800
@@ -104,7 +104,7 @@
     2 = One line per sample, elapsed time, temperature in C
     3 = Same as #2, except temperature is in F
 .PP
-#2 and #3 have the data seperated by tabs, suitable for import into a 
+#2 and #3 have the data separated by tabs, suitable for import into a 
 spreadsheet or other graphing software.
 .PP
 The format string uses strftime tokens plus 5 special ones for
diff -ruN digitemp-3.6.0.dfsg.1.orig/src/digitemp.c digitemp-3.6.0.dfsg.1/src/digitemp.c
--- digitemp-3.6.0.dfsg.1.orig/src/digitemp.c	2008-08-27 21:42:48.000000000 -0700
+++ digitemp-3.6.0.dfsg.1/src/digitemp.c	2012-12-08 13:54:16.617161520 -0800
@@ -196,8 +196,8 @@
   printf(BANNER_2);
   printf(BANNER_3, dtlib );		         /* Report Library version */
   printf("\nUsage: digitemp [-s -i -I -U -l -r -v -t -a -d -n -o -c]\n");
-  printf("                -i                            Initalize .digitemprc file\n");
-  printf("                -I                            Initalize .digitemprc file w/sorted serial #s\n");
+  printf("                -i                            Initialize .digitemprc file\n");
+  printf("                -I                            Initialize .digitemprc file w/sorted serial #s\n");
   printf("                -w                            Walk the full device tree\n");
   printf("                -s /dev/ttyS0                 Set serial port\n");
   printf("                -l /var/log/temperature       Send output to logfile\n");
@@ -218,7 +218,7 @@
   printf("\nLogfile formats:  1 = One line per sensor, time, C, F (default)\n");
   printf("                  2 = One line per sample, elapsed time, temperature in C\n");
   printf("                  3 = Same as #2, except temperature is in F\n");
-  printf("        #2 and #3 have the data seperated by tabs, suitable for import\n");
+  printf("        #2 and #3 have the data separated by tabs, suitable for import\n");
   printf("        into a spreadsheet or other graphing software.\n");
   printf("\n        The format string uses strftime tokens plus 5 special ones for\n");
   printf("        digitemp - %%s for sensor #, %%C for centigrade, %%F for fahrenheit,\n");
@@ -2432,7 +2432,7 @@
 
   if( argc <= 1 )
   {
-    fprintf(stderr,"Error! Not enough arguements.\n\n");
+    fprintf(stderr,"Error! Not enough arguments.\n\n");
     usage();
     return -1;
   }
diff -ruN digitemp-3.6.0.dfsg.1.orig/src/digitemp.h digitemp-3.6.0.dfsg.1/src/digitemp.h
--- digitemp-3.6.0.dfsg.1.orig/src/digitemp.h	2008-08-27 21:34:17.000000000 -0700
+++ digitemp-3.6.0.dfsg.1/src/digitemp.h	2012-12-08 13:54:16.617161520 -0800
@@ -19,7 +19,7 @@
    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
    ----------------------------------------------------------------------- */
 #define BANNER_1     "DigiTemp v3.5.0 Copyright 1996-2007 by Brian C. Lane\n"
-#define BANNER_2     "GNU Public License v2.0 - http://www.digitemp.com\n"
+#define BANNER_2     "GNU General Public License v2.0 - http://www.digitemp.com\n"
 #define BANNER_3     "Compiled for %s\n\n"
 
 #define OPT_INIT     0x0001
